rocketpy/Rocket.py

```diff
@@ -888,6 +891,14 @@ def allInfo(self):
         self.staticMargin()
         self.powerOnDrag()
         self.powerOffDrag()
+        self.thrustToWeight()
+
+        #ax = plt.subplot(415)
+        #ax.plot(  , self.rocket.motor.thrust()/(self.env.g() * self.rocket.totalMass()))
+        #ax.set_xlim(0, self.rocket.motor.burnOutTime)
+        #ax.set_xlabel("Time (s)")
+        #ax.set_ylabel("Thrust/Weight")
+        #ax.set_title("Thrust-Weight Ratio")
```

Comment:
Any reason for these lines to be here for the future?

Reply:
these lines would help us in editing figure size for Thrus/Weight plot